src/frontend/components/DashboardVideoLive/index.spec.tsx

```diff
@@ -249,6 +250,74 @@ describe('components/DashboardVideoLive', () => {
     );
 
     screen.getByRole('button', { name: /show live/i });
-    screen.getByRole('button', { name: /stop streaming/i });
+    screen.getByRole('button', { name: /pause ⏸/i });
+  });
+
+  it('polls the video when live state is STOPPING', async () => {
+    fetchMock.mock(
+      `/api/videos/${video.id}/`,
+      JSON.stringify({ ...video, live_state: liveState.STOPPING }),
+    );
+
+    const { rerender } = render(
+      wrapInIntlProvider(
+        wrapInRouter(
+          <Suspense fallback="loading...">
+            <DashboardVideoLive
+              video={{ ...video, live_state: liveState.STOPPING }}
+            />
+          </Suspense>,
+        ),
+      ),
+    );
+
+    screen.getByText(
+      'Pause live streaming, please wait a few moments to be able to resume it.',
+    );
+    expect(fetchMock.called()).not.toBeTruthy();
+
+    // First backend call: the live is still stopping
+    jest.advanceTimersByTime(1000 * 15 + 200);
+    await waitFor(() => expect(fetchMock.called()).toBe(true));
+
+    expect(fetchMock.lastCall()![0]).toEqual(`/api/videos/${video.id}/`);
+    expect(fetchMock.lastCall()![1]!.headers).toEqual({
+      Authorization: 'Bearer cool_token_m8',
+    });
+    screen.getByText(
+      'Pause live streaming, please wait a few moments to be able to resume it.',
```

Comment:
Do we have a state in which we tell the user that he can resume the streaming ?

Reply:
Yes, it's `PAUSED`

Reply:
Ok, I was searching a dedicated message for resuming a live, but I understand it's the same as `IDLE`

Reply:
Yes I kept the same `start streaming` button. Do you think we should the button label ?

Reply:
Yes, maybe it would be clearer for the user if we set the start button label to `Resume streaming` ?

Reply:
Done